icgc-argo/argo-data-submission
# Download aspera

To test:
```
nextflow tests/checker.nf -params-file tests/test-job-aspera.json
```

Test should download file using Aspera and output encrypted file.

`test-job-aspera.json` contents:
```
{
    "project" : "TEST",
    "input_file" : "/aspera-test-dir-large/100MB",  ### File to download
    "EGAF" : "EGAF000001" ### EGAF id for file
}
```

Additional info:
```
###Aspera valid credentials needed. Is preset in tests/nextflow.config
export ASCP_SCP_HOST="demo.asperasoft.com"
export ASCP_SCP_USER="aspera" 
export ASPERA_SCP_PASS="demoaspera"
```
